include/cutlass/arch/arch.h

```diff
@@ -97,6 +97,14 @@ struct Sm90 {
   static int const kMinComputeCapability = 90; 
 };
 
+#if defined(CUTLASS_ENABLE_SYCL)
```

Comment:
[Question]
Do we need to have `#if defined(CUTLASS_ENABLE_SYCL)` even around `struct` declarations ?